pkg/controller/disruption/disruption.go

```diff
@@ -294,6 +299,16 @@ func (dc *DisruptionController) getScaleController(controllerRef *metav1.OwnerRe
 	scale, err := dc.scaleNamespacer.Scales(namespace).Get(gr, controllerRef.Name)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			// The IsNotFound error can mean either that the resource does not exist,
```

Comment:
This code path seems to only affect the maxUnavailable code path, which I think is sensible based on the discussion in the associated issue.